### ISSUE TITLE
Screen Readers: Global: The currently active tab in the bottom tab bar not announced as such (mWeb)

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76956]
## Action Performed:
Prerequisite:
- User is logged in

1. Using Android Chrome + TalkBack or iOS Safari + VoiceOver, open the URL: https://new.expensify.com/
2. Swipe right to focus on the "Workspaces" tab in the bottom tab bar and double-tap to activate it
3. Swipe to focus the active tab and note how the element is announced.

## Expected Result:
The currently selected tab should be announced as such.

## Actual Result:
The currently active tab in the bottom tabbar is not announced as such.

Note: It applies to all bottom controls; the selected state is not announced.

HTML:
<button aria-label="Workspaces" role="button" data-selection-scrapper-hidden-element="true" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-1awozwy r-6koalj r-1pi2tsx r-1777fci r-n7gxbd r-9ffhgg r-lrvibr" type="button" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx"><div class="css-g5y9jx" data-element="View" data-source-file="index.tsx"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) viewBox="0 0 20 20" width="24" height="24" hovered="false" pressed="false" fill="#03D47C"><path d="M1 3a2 2 0 0 1 2-2h6a2 2 0 0 1 2 2v2h6a2 2 0 0 1 2 2v10a2 2 0 0 1-2 2H8v-3a1 1 0 0 0-1-1H5a1 1 0 0 0-1 1v3H3a2 2 0 0 1-2-2zm2 1a1 1 0 1 1 2 0 1 1 0 0 1-2 0m0 4a1 1 0 1 1 2 0 1 1 0 0 1-2 0m1 3a1 1 0 1 0 0 2 1 1 0 0 0 0-2m3-7a1 1 0 1 1 2 0 1 1 0 0 1-2 0m1 3a1 1 0 1 0 0 2 1 1 0 0 0 0-2m-1 5a1 1 0 1 1 2 0 1 1 0 0 1-2 0m5-5a1 1 0 1 0 0 2 1 1 0 0 0 0-2m3 1a1 1 0 1 1 2 0 1 1 0 0 1-2 0m1 3a1 1 0 1 0 0 2 1 1 0 0 0 0-2m-1 5a1 1 0 1 1 2 0 1 1 0 0 1-2 0m-3-5a1 1 0 1 0 0 2 1 1 0 0 0 0-2m-1 5a1 1 0 1 1 2 0 1 1 0 0 1-2 0" style="fill-rule: evenodd;"></path></svg></div></div><div dir="auto" class="css-146c3p1 r-dnmrzs r-1udh08x r-1udbk01 r-3s2u2q r-1iln25a" data-element="Text" data-source-file="index.tsx" style="color: rgb(231, 236, 233); font-size: 11px; text-align: center; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 700; margin-top: 6px; line-height: 14px;">Workspaces</div></button>

## Area issue was found in:
Workspace-Reports

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Implement an accessible tabpanel with aria-selected to announce the state.
https://www.w3.org/WAI/ARIA/apg/patterns/tabs/

## Environment(s)
Windows 11 Home - Chrome

## Screenshots/Videos
![Image](https://github.com/user-attachments/assets/d8e7dc8b-ad09-4362-a446-156123edb3f6)

https://github.com/user-attachments/assets/2a0f49ad-be6c-4043-b5dc-f600a71e79db

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6999525
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by rushatgabhane** [comment]
duplicate of 
- #77536             
- #77562
